Updating securing apps and authorization documentation based on the adapter deprecation/removal plan (#21177)
keycloak/keycloak: docs/documentation/securing_apps/topics/overview/getting-started.adoc

=== Getting Started

The link:{quickstartRepo_link}[{quickstartRepo_name}] provides examples about how to secure applications and services
using different programming languages and frameworks. By going through their documentation and codebase, you will
understand the bare minimum changes required in your application and service in order to secure it with {project_name}.

> You can also find in the following sections recommendations for trusted and well-known client-side implementations for both OpenID

Reviewer: ```suggestion
Also, see the following sections for recommendations for trusted and well-known client-side implementations for both OpenID
```
